src/Microsoft.Health.Fhir.Stu3.Web/appsettings.json

```diff
@@ -39,7 +39,8 @@
         "MaximumNumberOfConcurrentJobsAllowed": 1,
         "JobHeartbeatTimeoutThreshold": "00:10:00",
         "JobPollingFrequency": "00:00:10",
-        "SupportedDestinations": [ "" ]
+        "MaxItemCountPerQuery": 100,
```

Comment:
Should be `MaximumNumberOfResourcesPerQuery` to match with the field in `ExportJobConfiguration`

Reply:
Thanks for catching. I missed that.